### ISSUE TITLE
Changing media folder does not work

### ISSUE THREAD

**Issue by oorzaak** [joomla/joomla-cms#35689]
### Steps to reproduce the issue
Using ftp or a file browser, create a folder, e.g. "myfolder", in the site root and upload a test image to this folder (make sure to upload it to this folder only)

Go to Content > Media > Options

On the Media tab, change "Path to File Folder" and "Path to Image Folder" to point at "myfolder"

Then check in Content > Media if you can see your test image that is inside "myfolder"

### Expected result
I expect to see my test image in "myfolder", like I am used to see it in Joomla 3

### Actual result
I see the content of the Joomla default media folder "images"

### System information (as much as possible)
Joomla 4.0.3, php 7.4.23 

### Additional comments
In the database I can see that my folder settings are saved. So it seems to me that this setting is ignored.

**Comment by brianteeman** [comment]
Yes it is ignored and that is a bug - you should make the change in the FileSystem - Local plugin

![image](https://user-images.githubusercontent.com/1296369/135052073-54c45fa4-858a-4f43-bfb2-68d89227b3b6.png)

**Comment by oorzaak** [comment]
Thx for your reply, solves the problem indeed.

So it seems that at present there are 2 positions where you can change this setting. Will that be reduced to one and if so, which will be the 'final' one?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35689">issues.joomla.org/tracker/joomla-cms/35689</a>.</sub>

**Comment by brianteeman** [comment]
refereence https://github.com/joomla/joomla-cms/issues/31568

**Comment by sozzled** [comment]
This has been discussed on _[The Joomla Forum™](https://forum.joomla.org/viewtopic.php?f=811&t=988915)_ and referenced here at https://github.com/joomla/joomla-cms/issues/23972, https://github.com/joomla/joomla-cms/issues/31568, https://github.com/joomla/joomla-cms/issues/35505 and https://github.com/joomla/joomla-cms/issues/35689.

Intuitively, if the facility used in J! 3.x (which works)—which involves going to the **Media** component options to display a form where you can make changes to the paths—has been transferred to J! 4 (where it does not work), then this is confusing.  Either the **Media** component form should not display places to set the image folder/file paths or the failure to use that mechanism should be classed as a bug.  Although this may affect a small percentage of website owners who want to change the default images location to something else, unless people discover that the "options" feature doesn't work for themselves and they search the internet to find out how to do what they want by changing the _Filesystem - Local_ plugin, they will be puzzled.

I searched for documentation about the _Filesystem - Local_ plugin but I can't locate anything at docs.joomla.org.

**Comment by worech** [comment]
Still not solved, the Filesystem - Local plugin use is a very confusing way.
I am a newbie here - where to set this problem as a bug?
Thanks, guys.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35689">issues.joomla.org/tracker/joomla-cms/35689</a>.</sub>

**Comment by Webdongle** [comment]
Also with the plugin the extra values are not observed if insert image is accessed via the toolbar icon.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35689">issues.joomla.org/tracker/joomla-cms/35689</a>.</sub>

**Comment by sozzled** [comment]
I think we've dealt with that on the forum, @Webdongle.  See https://forum.joomla.org/viewtopic.php?f=808&t=997168#p3673739.  There's a quirk with _JCE Editor_ but _TinyMCE_ works OK.

**Comment by Webdongle** [comment]
No in the tracker it hasn't
